setup.cfg

```diff
@@ -0,0 +1,40 @@
+[metadata]
+name = sktm
+description = "Sonic Kernel Testing"
+long_description = file: README.md
+version = 1
+author = Red Hat, Inc.
+license = GPLv2+
+
+[options]
+# Automatically find all files beneath the skt directory and include them.
+packages = find:
+# Parse the MANIFEST.in file and include those files, too.
+include_package_data = True
+# Let pip install dependencies automatically.
+install_requires = dateutils
+                   enum34
+                   jenkinsapi
+                   junit_xml
+                   flake8
+                   pycodestyle!=2.4.0
+                   pylint
```

Comment:
Wait, we don't need flake8, pycodestyle, or pylint packages for sktm to function. These are purely our CI requirements, and, it appears to me, shouldn't be here. Can you fix that? And same for skt?

Reply:
Maybe we can use requirements-dev.txt for this. Then install dev dependencies via `pip install -r requirements-dev.txt`.

Reply:
Fixed.